backtrace-library/src/main/java/backtraceio/library/models/json/BacktraceAttributes.java

```diff
@@ -199,4 +197,27 @@ private void convertAttributes(Map<String, Object> attributes) {
             }
         }
     }
+
+    public String getApplicationName() {
+        return this.context.getApplicationInfo().loadLabel(this.context
+                .getPackageManager()).toString();
+    }
+
+    public String getApplicationVersionOrEmpty() {
+        try {
+            return this.context.getPackageManager()
+                    .getPackageInfo(this.context.getPackageName(), 0).versionName;
+        } catch (PackageManager.NameNotFoundException e) {
+            BacktraceLogger.e(LOG_TAG, "Could not resolve application version");
+            e.printStackTrace();
+        }
+        return "";
+    }
+
+    public Map<String, Object> getAllBacktraceAttributes() {
```

Comment:
I think function name doesn't tell you exactly what it does. It doesn't return ALL attributes. It doesn't even return built-in attributes.  Wouldn't it be better if call this function: `getApplicationInformationAttributes` or something like this?

Reply:
This is `BacktraceAttributes` class and it will return all `BacktraceAttributes`. But since it's already within `BacktraceAttributes` class, we can rename it to `getAllAttributes`. Does that work @konraddysput ?

Reply:
Yeah, that works - but one sec. I just recognize this is a public function. Is it something that we added in this pull request, or is it something that we added in the previous version?

Reply:
@konraddysput we are adding this in this pull request, is it okay?

Reply:
Wouldn't it be better if this function will be internal instead? If it's impossible to make it private/internal public is also fine.